Tasks/NuGetToolInstallerV1/package.json

```diff
@@ -24,8 +24,9 @@
     "homepage": "https://github.com/Microsoft/azure-pipelines-tasks#readme",
     "dependencies": {
         "@types/mocha": "^5.2.7",
-        "@types/node": "^10.17.0",
+        "@types/node": "^16.11.39",
         "@types/uuid": "^8.3.0",
+        "azure-pipelines-task-lib": "^4.1.0",
```

Comment:
Why do we need a new package dependency?

Reply:
I have removed the dependency from azure-pipelines-task-lib but had to make an edit in make.json.
Requesting you to kindly review all the changes and approve if it's fine.